jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/traveler/TimTransmogrifierTest.java

```diff
@@ -27,262 +30,256 @@
+
+        TravelerInputData fakeTID = new TravelerInputData();
+        fakeTID.setRequest(inputServiceRequest);
+
+        ObjectNode encodableTID = JsonUtils.toObjectNode(JsonUtils.toJson(fakeTID, false));
+
+        SerialId staticSerialId = new SerialId();
+        staticSerialId.setStreamId("6c33f802-418d-4b67-89d1-326b4fc8b1e3");
+
+        OdeMsgMetadata staticOdeMsgMetadata = new OdeMsgMetadata();
+
+        staticOdeMsgMetadata.setSchemaVersion(Integer.parseInt(schemaVersion));
+
+        String actualXML = TimTransmogrifier.convertToXml(actualASD, encodableTID, staticOdeMsgMetadata, staticSerialId);
+        String expected = String.format("<OdeAsn1Data><metadata><payloadType>us.dot.its.jpo.ode.model.OdeAsdPayload</payloadType><serialId><streamId>6c33f802-418d-4b67-89d1-326b4fc8b1e3</streamId><bundleSize>1</bundleSize><bundleId>0</bundleId><recordId>0</recordId><serialNumber>0</serialNumber></serialId><odeReceivedAt>%s</odeReceivedAt><schemaVersion>%s</schemaVersion><maxDurationTime>0</maxDurationTime><sanitized>false</sanitized><request><sdw><serviceRegion><nwCorner><latitude>42.537903</latitude><longitude>-83.477903</longitude></nwCorner><seCorner><latitude>42.305753</latitude><longitude>-82.842753</longitude></seCorner></serviceRegion><ttl>thirtyminutes</ttl><deliverystart>2017-06-01T17:47:11-05:00</deliverystart><deliverystop>2018-03-01T17:47:11-05:15</deliverystop></sdw><rsus/></request><encodings><encodings><elementName>MessageFrame</elementName><elementType>MessageFrame</elementType><encodingRule>UPER</encodingRule></encodings><encodings><elementName>Ieee1609Dot2Data</elementName><elementType>Ieee1609Dot2Data</elementType><encodingRule>COER</encodingRule></encodings><encodings><elementName>AdvisorySituationData</elementName><elementType>AdvisorySituationData</elementType><encodingRule>UPER</encodingRule></encodings></encodings></metadata><payload><dataType>us.dot.its.jpo.ode.plugin.j2735.DdsAdvisorySituationData</dataType><data><AdvisorySituationData><dialogID>156</dialogID><seqID>5</seqID><groupID>00000000</groupID><requestID>7876BA7F</requestID><recordID>00000000</recordID><timeToLive>1</timeToLive><serviceRegion><nwCorner><lat>425379030</lat><long>-834779030</long></nwCorner><seCorner><lat>423057530</lat><long>-828427530</long></seCorner></serviceRegion><asdmDetails><asdmID>7876BA7F</asdmID><asdmType>2</asdmType><distType>03</distType><startTime><year>2017</year><month>6</month><day>1</day><hour>17</hour><minute>47</minute></startTime><stopTime><year>2018</year><month>3</month><day>1</day><hour>17</hour><minute>47</minute></stopTime><advisoryMessage><Ieee1609Dot2Data><protocolVersion>3</protocolVersion><content><unsecuredData><MessageFrame><messageId>31</messageId><value><TravelerInformation/></value></MessageFrame></unsecuredData></content></Ieee1609Dot2Data></advisoryMessage></asdmDetails></AdvisorySituationData></data></payload></OdeAsn1Data>", DateTimeUtils.now(), schemaVersion);
```

Comment:
note: While I double-checked that I didn't change any intended behaviors, I had to instantiate and set a GeoServiceRegion when I removed JMockit's mocking. I'm pretty sure by removing the mocking, we've shown the intended behavior hidden by JMockit before. I would absolutely love some extra eyes on this and the other tests in this file to make sure the tests still cover the expected behavior